api/controllers/calendarEntry.controller.ts

```diff
@@ -75,18 +71,28 @@ export const createCalendarEntry = async (
 export const getCalendarEntries = async (
   _req: Request,
   res: Response,
-  _next: NextFunction
+  _next: NextFunction,
 ) => {
-  const entries = await CalendarEntry.find();
+  const entries: CalendarEntry = await CalendarEntry.find();
   res.status(200).json(entries);
 };
 
 export const getCalendarEntry = async (
   req: Request,
   res: Response,
-  next: NextFunction
+  _next: NextFunction,
 ) => {
   const { id } = req.params;
   const entry = await CalendarEntry.findById(id);
   res.status(200).json(entry);
 };
+
+export const deleteCalendarEntry = async (
+  req: Request,
+  res: Response,
+  _next: NextFunction,
+) => {
+  const { id } = req.params;
+  await CalendarEntry.deleteOne({ _id: id });
+  res.status(200).json();
```

Comment:
I think we should have some sort of basic error handling here, so that if `deleteOne` fails, we don't send a 200. I guess that goes for the whole app. maybe this would be a whole other ticket

Reply:
yeah let's do this in a follow on ticket